src/kibana/styles/main.less

```diff
@@ -359,6 +359,12 @@ saved-object-finder {
     font-size: 0.85em;
   }
 
+  div.finder-form-options{
```

Comment:
Use the LESS nesting syntax to make this part of div.finder-form on line 351: https://github.com/BigFunger/kibana/blob/fix-3899/src/kibana/styles/main.less#L351

```
  div.finder-form {
    position: relative;

    &-options {
      .flex-parent();
      .flex-direction(row);
      .justify-content(flex-end);
    }
  }
```